package mirrtalk.traffic.fp.receiver.positionprotocol;                                                   
message CiTyWLPosition{
        required string imei = 1; //车辆ID
        required fixed32 lat = 2;//维度，放大1000000倍
        required fixed32 lon = 3;//经度，放大1000000倍
        required int32 speed = 4; //速度，放大100倍
        required int32 direction = 5; //角度
        required int32 collectTime = 6;//GPS点时间,unix时间戳
}
 
message CiTyWLGpsData {
        required int32 gpsCount = 1;
        repeated CiTyWLPosition  position = 2;
}
